build/vsts_build.yaml

```diff
@@ -529,33 +522,16 @@ jobs:
     # The Microsoft.DotNet.Build.Tasks.Feed package includes NuGet.Common 4.9.0.6 and a binding redirection in Microsoft.DotNet.Build.Tasks.Feed.dll.config but the binding redirection is not processed.
     # This would probably solve it:  https://github.com/Microsoft/msbuild/issues/1309
   - task: CmdLine@2
-    displayName: "Publish the packages to the .NET Core build asset registry (BAR)"
+    displayName: "Publish to the .NET Core build asset registry (BAR)"
     inputs:
-      script: dotnet msbuild ..\build\publish.proj /t:PublishPackagesToBuildAssetRegistry /p:NuGetClientNupkgsDirectoryPath=$(Build.Repository.LocalPath)\artifacts\$(NupkgOutputDir) /p:FeedUrl=$(DotNetFeedUrl) /p:BuildId=$(Build.BuildNumber) /p:RepoBranch=$(Build.SourceBranchName) /p:RepoCommit=$(Build.SourceVersion) /p:RepoUri=$(Build.Repository.Uri) /p:ManifestFilePath=$(ManifestFilePath) /p:AccountKey=$(dotnetfeed-storage-access-key-1)
+      script: dotnet msbuild ..\build\publish.proj /t:PublishToBuildAssetRegistry /p:NuGetClientNupkgsDirectoryPath=$(Build.Repository.LocalPath)\artifacts\$(NupkgOutputDir) /p:FeedUrl=$(DotNetFeedUrl) /p:BUILD_BUILDNUMBER=$(Build.BuildNumber) /p:BUILD_SOURCEBRANCH=$(Build.SourceBranchName) /p:BUILD_SOURCEVERSION=$(Build.SourceVersion) /p:BUILD_REPOSITORY_URI=$(Build.Repository.Uri) /p:ArtifactsLogDir=$(Build.Repository.LocalPath)\artifacts\manifests\ /p:MaestroApiEndpoint=$(MaestroApiEndpoint) /p:MaestroAccessToken=$(MaestroAccessToken)
```

Comment:
You don't need to specify some of these properties and some you can just use their value directly in the publish.proj file, instead of receiving them as parameter. Specifically, the FeedUrl property you don't need anymore. The /p:BUILD_BUILDNUMBER property you can just reference it directly in publish.proj using `$(BUILD_BUILDNUMBER)`.

Reply:
so, while I know that MSBuild make environment variables into properties, I don't know if it's good practise to be more explicit about what's actually being used, or to just go ahead and use properties not explicitly passed in the script.  Does my team have any preferences?

Reply:
Fine for me.